tests/app/main/test_briefs.py

```diff
@@ -120,6 +129,7 @@ def test_clarification_question_checks_supplier_is_eligible(self, data_api_clien
         self.login()
         data_api_client.get_brief.return_value = api_stubs.brief(status='live')
         data_api_client.get_brief.return_value['briefs']['frameworkName'] = 'Digital Outcomes and Specialists'
+        data_api_client.get_brief.return_value['briefs']['lot'] = 'digital-specialists'
```

Comment:
@TheDoubleK Doing this reveals an interesting discrepancy - a serialization of a `Brief` includes both a `lot` and `lotSlug` attribute, and I have been using the `lot`. However, `lotSlug` _appears_ to be newer and preferred as `lot` isn't included everywhere (including in the stub). Guess I'll switch to using lotSlug?